deploying-monitoring-redis-oke.md

```diff
@@ -0,0 +1,257 @@
+---
+title: Deploying and monitoring a Redis cluster to Oracle Container Engine (OKE)
+parent: [tutorials]
+thumbnail: assets/redis-dashboard.png
+tags: [backend, oci]
+date: 2021-12-15 13:34
+description: Redis, Prometheus, OKE, oh my!
+MRM: WWMK211213P00053
+author: ali-mukadam
```

Comment:
I think I may have given confusing instructions, but when the author exists in _authors.yml, the front matter key should be configured as `author: AUTHOR_SLUG`

Reply:
✅